### DISCUSSION TITLE
Blue pause symbol without a paper plane?

### DISCUSSION THREAD

**Discussion by Qhilm** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1347] [Q&A]
Hello, my Mac is somehow not charging, it shows a blue pause symbol without any paper plane:

<img width="386" alt="image" src="https://github.com/user-attachments/assets/fc1e2a0c-c3d8-4eea-b62e-60ade5077394">

Heat protection is off, my charger can deliver 40W but the MacBook currently only consumes 10-12W, the battery does not show up in the visual flow. The same charger was perfectly capable of charging the MacBook to 100% a couple of weeks ago.

It sounds as if the macOS was overriding AlDente.  I reinstalled the helper, but it did not help.

If I click "charge to full now" in macOS, it works, it starts charging at 40W.

Should I send the debug files somewhere?

macOS 15.0.1, AlDente Pro 1.28.5.

**Comment by MatthiasKerbl** [comment]
> Hello, my Mac is somehow not charging, it shows a blue pause symbol without any paper plane:
> 
> <img alt="image" width="386" src="https://private-user-images.githubusercontent.com/3350433/374425433-fc1e2a0c-c3d8-4eea-b62e-60ade5077394.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgzODQ3NjQsIm5iZiI6MTcyODM4NDQ2NCwicGF0aCI6Ii8zMzUwNDMzLzM3NDQyNTQzMy1mYzFlMmEwYy1jM2Q4LTRlZWEtYjYyZS02MGFkZTUwNzczOTQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAwOCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMDhUMTA0NzQ0WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTc1ZGI5YjU5MzhjMWEwNzViOWY0MDNiZjE1ODY5MmQzMDc0YzYyNTY5MmQ1OTY2OTViNWQ3NjBiNTc0ZjI0OSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.S122ou9rn2t4zZ0xtOF5u8zu1gcBvyzY8VYXRySrqPk">
> Heat protection is off, my charger can deliver 40W but the MacBook currently only consumes 10-12W, the battery does not show up in the visual flow. The same charger was perfectly capable of charging the MacBook to 100% a couple of weeks ago.
> 
> It sounds as if the macOS was overriding AlDente. I reinstalled the helper, but it did not help.
> 
> If I click "charge to full now" in macOS, it works, it starts charging at 40W.
> 
> Should I send the debug files somewhere?
> 
> macOS 15.0.1, AlDente Pro 1.28.5.

Hi @Qhilm,

Do you have Apple's Optimized Battery charging disabled?

Best,
Matthias

**Comment by Qhilm** [comment]
Thanks, somehow macOS Sequoia re-enabled this setting, it was off before. Thanks!
